packages/core/src/asciidoctor-core-api.js

```diff
@@ -3047,3 +3047,381 @@ Opal.Asciidoctor.SyntaxHighlighterBase = SyntaxHighlighterBase
+}
+
+/**
+ * Get the colspan of this {@link Cell} node
+ * @memberof Cell
+ * @returns {number}  An Integer of the number of columns this cell will span (default undefines)
+ */
+Cell.prototype.getColspan = function () {
+  var colspan = this.colspan
+  return colspan === Opal.nil ? undefined : colspan
+}
+
+/**
+ * set the colspan of this {@link Cell} node
+ * @param {string} value
+ * @returns{string} the new colspan value
+ * @memberof Cell
+ */
+Cell.prototype.setColspan = function (value) {
+  return this['$colspan='](value)
+}
+
+/**
+ * get the rowspan of this {@link Cell} node
+ * @memberof Cell
+ * @returns {number}  An Integer of the number of rows this cell will span (default undefined)
+ */
+Cell.prototype.getRowspan = function () {
+  var rowspan = this.rowspan
+  return rowspan === Opal.nil ? undefined : rowspan
+}
+
+/**
+ * set the rowspan of this {@link Cell} node
+ * @param {string} value
+ * @returns{string} the new rowspan value
+ * @memberof Cell
+ */
+Cell.prototype.setRowspan = function (value) {
+  return this['$rowspan='](value)
+}
+
+/**
+ * Get the content of the cell
+ * This method should not be used for cells in the head row or that have the literal style.
+ */
+Cell.prototype.getContent = function () {
+  var content = this['$content']()
+  return content === Opal.nil ? undefined : content
+}
+
+/**
+ * Get the text of the cell
+ * @returns {string}
+ */
+Cell.prototype.getText = function () {
+  var text = this['$text']()
+  return text === Opal.nil ? undefined : text
+}
+
+/**
+ * Get the source of the cell
+ * @returns {string}
+ */
+Cell.prototype.getSource = function () {
+  var source = this['$source']()
+  return source === Opal.nil ? undefined : source
+}
+
+/**
+ * Get the lines of the cell
+ * @returns {Array}
+ */
+Cell.prototype.getLines = function () {
+  var lines = this['$lines']()
+  return lines === Opal.nil ? undefined : lines
+}
+
+/**
+ * Get the lineno of the cell
+ * @returns {string}
+ */
+Cell.prototype.getLineno = function () {
+  var lineno = this['$lineno']()
+  return lineno === Opal.nil ? undefined : lineno
+}
+
+/**
+ * Get the file of the cell
+ * @returns {string}
+ */
+Cell.prototype.getFile = function () {
+  var file = this['$file']()
+  return file === Opal.nil ? undefined : file
+}
+
+/**
+ * Get the style of the cell
+ * @returns {string}
+ */
+Cell.prototype.getStyle = function () {
+  var style = this['$style']()
+  return style === Opal.nil ? undefined : style
+}
+
+/**
+ * Get the column of this cell
+ * @returns {Column}
+ */
+Cell.prototype.getColumn = function () {
+  var column = this.$column()
+  return column === Opal.nil ? undefined : column
+}
+
+/**
+ * Get the width of this cell
+ * @returns {string|undefined}
+ */
+Cell.prototype.getWidth = function () {
+  return this.getAttribute('width')
+}
+
+/**
+ * Get the colpcwidth of this cell
+ * @returns {string|undefined}
+ */
+Cell.prototype.getColpcwidth = function () {
```

Comment:
I think we should use long form (ie. `getColumnPercentageWidth`) because the rest of the API is using long form.
At some point, we might introduce alias:

- `getColPercentWidth`
- `getColPcWidth`

But we need to be careful because users will expect `Col` to be equivalent to `Col` in other functions as well.

Reply:
Changed name to getColumnPercentageWidth. No alias defined by now

Reply:
Thanks!